docs/data/data-grid/filtering/filtering.md

```diff
@@ -368,13 +368,16 @@ The values used by the quick filter are obtained by splitting with space.
 If you want to implement a more advanced logic, the `<GridToolbarQuickFilter/>` component accepts a prop `quickFilterParser`.
 This function takes the string from the search text field and returns an array of values.
 
+If you control the `quickFilterValues` either by controlling `filterModel` or with the initial state, you should provide a `quickFilterFormatter` which will be used to update the content of the input.
```

Comment:
I didn't understand this part. There's a default value formatter, the user doesn't need to pass one.

Reply:
I tried to be more explicit